### PR TITLE
add a ci for the first release.

### PR DESCRIPTION
**Pull request by utam0k**

### CODE REVIEW

**Comment by utam0k** [comment]
https://github.com/utam0k/youki/runs/4129908393

File: .github/workflows/releease.yml [hunk 1]
```diff
@@ -0,0 +1,122 @@
+name: Release
```
Comment (yihuaf) [inline .github/workflows/releease.yml:1]: mis-spell the file name `releease.yml` -> `release.yml`

Comment (utam0k) [reply .github/workflows/releease.yml]: Thanks! I have fixed it.

**Comment by utam0k** [comment]
@Furisto @yihuaf @tsturzl 
Can I have your review?

**Review by yihuaf — APPROVED** [review]
LGTM assume this is tested. On a side note, we should look into how to share some of the steps in the CI. We have accumulated a decent number of code duplication.

**Comment by utam0k** [comment]
> GTM assume this is tested. On a side note, we should look into how to share some of the steps in the CI. We have accumulated a decent number of code duplication.

I've considered this before but gave up because it seemed too difficult.

**Pull request merged**